src/sentry/analytics/events/admin_request.py

```diff
@@ -0,0 +1,16 @@
+from sentry import analytics
+
+
+class InviteRequestSent(analytics.Event):
```

Comment:
@Neo-Zhixing So we should make a class called `InAppRequest` or something that has the attributes we want then have `InviteRequestSent` and the other events subclass it where each set their own `type` field

Reply:
I see.